apps/heft/src/templates/typescript.json

```diff
@@ -36,6 +36,16 @@
     // }
   ],
 
+  /**
+   * If true, will emit CommonJS module output to \"lib\" with the .cjs extension alongside (or instead of, if TSConfig specifies CommonJS) the default compilation output.
```

Comment:
```suggestion
   * If true, emit CommonJS module output to the folder specified in the tsconfig "outDir" compiler option with the .cjs extension alongside (or instead of, if TSConfig specifies CommonJS) the default compilation output.
```